xrm/index.d.ts

```diff
@@ -474,6 +549,179 @@ declare namespace Xrm
+          *                                                                             DateTime
+          *                                                                             Decimal
+          *                                                                             Double
+          *                                                                             EntityName
+          *                                                                             Integer
+          *                                                                             Lookup
+          *                                                                             Memo
+          *                                                                             Money
+          *                                                                             Owner
+          *                                                                             Picklist
+          *                                                                             String
+          *                                                                             State
+          *                                                                             Status
+          *                                                                             UniqueIdentifier
+          */
+        retrieveRecord( entityType: string, id: string, options: string ): Async.XrmPromise<Async.OfflineOperationSuccessCallbackDelegate, Async.ErrorCallbackDelegate>;
+
+        /**
+          * Retrieves a collection of entity records in mobile clients while working in the offline mode.
+          *
+          * @param    {string}    entityType    The logical name of the entity.
+          * @param    {string}    options       (Optional) The logical name of the enti
+          * @param    {string}    maxPageSize   (Optional) A positive number to indicates the number of entity records to be returned per page.
+          *                                     If you do not specify this parameter, the default value is passed as 5000.
+          *                                     If the number of records being retrieved is more than maxPageSize, an @odata.nextLink property
+          *                                     will be returned, and you can use the value of the @odata.nextLink property with a new GET
+          *                                     request to return next set of records.
+          *
+          * @return   {Async.XrmPromise} Returns an asynchronous promise.
+          *
+          * @remarks  A maximum of 5000 related records can be retrieved when using $expand.
+          *           Only $select option can be specified within $expand.
+          *           $skip query option is not supported.
+          *           Only the following attribute types are supported in offline mode: BigInt
```

Comment:
I tried to think of a way to do that but came up blank. Maybe putting them in a few lines instead of one per line will make it a bit easier to read?